### PR TITLE
Documentation updates [5372]

### PR DESCRIPTION
**Pull request by raquelalvarezbanos**
This PR adds some updates to the Fast-RTPS documentation for release 1.8.0

### CODE REVIEW

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/1697/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/1656/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/1668/)

File: README.md [hunk 1]
```diff
@@ -42,7 +43,7 @@ You can get either a binary distribution of *eprosima Fast RTPS* or compile the
 The latest, up to date binary release of *eprosima Fast RTPS* can be obtained from the <a href='http://www.eprosima.com'>company website</a>.
 
 ### Installation from Source
-To compile *eprosima Fast RTPS* from source, at least Cmake version 2.8.12 is needed.
+To compile *eprosima Fast RTPS* from source, at least Cmake version 2.8.12 and Boost 1.61 are needed.
```
Comment (LuisGP) [inline README.md]: Did somebody add boost dependency?

File: README.md [hunk 2]
```diff
@@ -52,13 +53,13 @@ Clone the project from GitHub:
 
 If you are on Linux, execute:
 
-    $ cmake -DTHIRDPARTY=ON ..
+    $ cmake ../ -DTHIRDPARTY=ON -DEPROSIMA_BUILD=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=install
```
Comment (LuisGP) [inline README.md]: If you want to install Fast-RTPS in the system without examples or testing, all these options aren't needed. Telling the user to call `cmake ..` should add another command before like `mkdir build && cd build`.

File: README.md [hunk 3]
```diff
@@ -52,13 +53,13 @@ Clone the project from GitHub:
...
     $ make
     $ make install
 
 If you are on Windows, choose your version of Visual Studio:
 
-    > cmake -G "Visual Studio 14 2015 Win64" -DTHIRDPARTY=ON ..
+    > cmake ../  -G "Visual Studio 14 2015 Win64" -DTHIRDPARTY=ON -DEPROSIMA_BUILD=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=installationpath
```
Comment (LuisGP) [inline README.md]: Same as Linux.

File: README.md (hunk 1 above)
Comment (raquelalvarezbanos) [reply README.md]: No, I think I just got confused with the README.md of Fast-RTPS-docs. Removed.

File: README.md (hunk 2 above)
Comment (raquelalvarezbanos) [reply README.md]: :+1:

File: README.md (hunk 3 above)
Comment (raquelalvarezbanos) [reply README.md]: :+1:

**Comment by MiguelCompany** [comment]
Merging directly as this does not have code changes

**Pull request merged**